core/controllers/collection_editor.py

```diff
@@ -90,6 +90,11 @@ def put(self, collection_id):
         _require_valid_version(version, collection.version)
 
         commit_message = self.payload.get('commit_message')
+
+        if len(commit_message) > feconf.MAX_LENGTH_OF_COMMIT_MESSAGES:
+            raise self.InvalidInputException(
+                'Commit messages must be at most 1000 characters.')
```

Comment:
...at most %s characters **long**.' % MAX_COMMIT_MESSAGE_LENGTH

Ditto elsewhere. (But, keep the message in the tests hardcoded.)

Reply:
Done